4-WebApp-your-API/4-3-AnyOrg/AppCreationScripts/sample.json

```diff
@@ -5,8 +5,79 @@
     "Client": "ASP.NET Core Web App",
     "Service": "ASP.NET Core Web API",
     "RepositoryUrl": "microsoft-identity-platform-aspnetcore-webapp-tutorial",
-    "Endpoint": "AAD v2.0"
+    "Endpoint": "AAD v2.0",
+    "Products": [ "azure", "dotnet", "azure-active-directory", "office-graph-api" ],
+    "Platform": "AspNetCore",
+    "Languages": [ "csharp" ],
+    "Library": [ "Microsoft.Identity.Web" ]
   },
+
+  /*
+    This section describes different regions of readme file
+  */
+  "ReadmeScenario": {
+    "IncludeFilePath": "../ReadmeFiles/ReadmeScenario.md",
+    "Image": "./ReadmeFiles/topology.png",
+    /* put additional notes, will be displayed right after image*/
+    "AdditionalNotesIncludeFilePath": ""
+  },
+
+  "ReadmePrerequirements": {
+    "FreeText": "",
+    "IncludeFilePath": ""
+  },
+
+  "ReadmeSetup": {
+    "FreeText": "",
+    "IncludeFilePath": "",
+    "CertificateOption": "1", /* 1 when a certificate can be used instead of secret*/
+    "CreateProjectIncludeFilePath": "",
+    "AppRegistrationIncludeFilePath": "",
+    "RunSampleIncludeFilePath": ""
+  },
+
+  /* It either can be a text or link to another readme file */
+  "ReadmeTroubleshooting": {
+    "IncludeFilePath": ""
+  },
+
+  /* It either can be a text or link to another readme file */
+  "ReadmeNextSteps": {
+    "FreeText": "",
+    "IncludeFilePath": ""
+  },
+
+  /* It either can be a text or link to another readme file */
+  "ReadmeContributing": {
+    "FreeText": "",
+    "IncludeFilePath": ""
+  },
+
+  /* It either can be a text or link to another readme file */
+  "ReadmeHowTheCodeWasCreated": {
+    "IncludeFilePath": "../ReadmeFiles/ReadmeHowTheCodeWasCreated.md"
+  },
+
+  /* It either can be a text or link to another readme file */
+  "ReadmeAboutTheCode": {
+    "IncludeFilePath": "../ReadmeFiles/ReadmeAboutTheCode.md"
+  },
+
+  /* It either can be a text or link to another readme file */
+  "ReadmeAzureDeploy": {
+    "IncludeFilePath": "../ReadmeFiles/ReadmeHowToDeploy.md"
```

Comment:
Could not find file 'D:\github\Azure-Samples\active-directory-aspnetcore-webapp-openidconnect-v2\4-WebApp-your-API\4-3-AnyOrg\ReadmeFiles\ReadmeAboutTheCode.md'.

Could not find file 'D:\github\Azure-Samples\active-directory-aspnetcore-webapp-openidconnect-v2\4-WebApp-your-API\4-3-AnyOrg\ReadmeFiles\ReadmeHowTheCodeWasCreated.md'.

Could not find file 'D:\github\Azure-Samples\active-directory-aspnetcore-webapp-openidconnect-v2\4-WebApp-your-API\4-3-AnyOrg\ReadmeFiles\ReadmeHowToDeploy.md'.